test/e2e/__snapshots__/ClientOptions.test.js.snap

```diff
@@ -31,5 +31,6 @@ Array [
 exports[`Client console.log liveReload enabled 1`] = `
 Array [
   "Hey.",
+  "[WDS] Live Reloading enabled.",
```

Comment:
why it is appears here?

Reply:
@evilebottnawi Previously, `inline` was disabled for this test, so the live reload message was not printed. Look at the change in `test-server.js`. I made it so that `options.inline` is not set automatically to `false` when `options.liveReload` exists. This test makes more sense if `inline` is enabled.